### DISCUSSION TITLE
Practicality thoughts and questions

### DISCUSSION THREAD

**Discussion by markdrichter-tw** [Green-Software-Foundation/sci#257] [SCI feedback]
I'll begin by applauding everyone involved with the SCI and the (I suppose lively) discussions and work to pull this together. Nice work! What I have here is some feedback as I think about its practical use. 

If I want to propose development teams begin integrating SCI rating into their software design, how can this be accomplished in the practical sense? Why do I ask? It seems to me that E, I, and M are quite difficult to derive. Without fairly robust, off-the-shelf models to generate this data I struggle to see how to use the rating formula.

I see a number of discussions on the board, but I couldn't suss out crisp answers. I'm sure that the working group has thought about this. I'd love to know what other folks think.

Cheers,

Mark Richter
Head of Cloud Ops
Thoughtworks

**Comment by Henry-WattTime** [comment]
Hello Mark,

Thanks for the kind words.

We've been putting considerable work into this effort, on two fronts. There's a project at the GSF to identify and track data sources for each term of the formula. The open source data project is where most of this is happening at the GSF. The GSF is also building out case studies (would you like to contribute one?!) so that data availability obstacles can be identified. We hope that demonstrating how a case study is build will show developers the practical steps in calculating a score.

We're also at the beginning of building out the ecosystem to support calculations like this. The specification . Making this data accessible and actionable is one of the primary challenges and goals of the GSF right now. In the future, once the data is easily available, we hope that developers could calculate their SCI score as part of the release process. 

Please let us know if you have any questions or would like to be connected with any of the groups. Would you like to join the group to discuss? 

Thanks! -Standards WG

**Comment by markdrichter-tw** [comment]
I am very interested in joining the group. FYI, I'll be on vacation the
first half of July and again for two weeks in the middle of August.

Mark Richter @ San Diego

**Comment by atg-abhishek** [comment]
Hope you're able to join some of the WG meetings @markdrichter-tw - if you have trouble finding the details, please feel free to reach out to @seanmcilroy29.

**Comment by markdrichter-tw** [comment]
If someone could send me a meeting invitation it would be helpful.

Mark Richter @ San Diego

**Reply by seanmcilroy29** [reply]
Invite sent
